src/Microsoft.ApplicationInsights.AspNet/RequestTrackingMiddleware.cs

```diff
@@ -50,6 +52,7 @@ public RequestTrackingMiddleware(RequestDelegate next, TelemetryClient client)
                     telemetry.ResponseCode = httpContext.Response.StatusCode.ToString();
                     telemetry.Success = httpContext.Response.StatusCode < 400;
                     telemetry.HttpMethod = httpContext.Request.Method;
+                    telemetry.Url = httpContext.Request.GetUri();
```

Comment:
@Sergei-Nikitin and I respectfully disagree :smile: and will try to convince you later tonight :facepunch: